hadoop-ozone/dist/src/main/compose/common/hadoop-test.sh

```diff
@@ -33,9 +33,15 @@ export OZONE_DIR=/opt/ozone
 # shellcheck source=/dev/null
 source "$COMPOSE_DIR/../testlib.sh"
 
-for HADOOP_VERSION in 2.7.3 3.1.2 3.2.2 3.3.1; do
+for HADOOP_VERSION in 2.7.3 3.1.2 3.2.2 3.3.5; do
   export HADOOP_VERSION
   export HADOOP_MAJOR_VERSION=${HADOOP_VERSION%%.*}
+  # Check if $HADOOP_VERSION starts with the prefix "3.3."
+  if [[ $HADOOP_VERSION == 3.3.* ]]; then
```

Comment:
Is there a relatively easy way to make this `$HADOOP_VERSION >= 3.3.*` ?

If not it should work at least for now, until Hadoop 3.4.0 is released.

Reply:
This is exactly what ChatGPT is for! I asked ChatGPT and it answered:


```
#!/bin/bash

# Extracting X, Y, and Z from HADOOP_VERSION
IFS='.' read -r -a version_parts <<< "$HADOOP_VERSION"
X=${version_parts[0]}
Y=${version_parts[1]}
Z=${version_parts[2]}

# Checking the condition and printing "hello world" if met
if [[ $X -eq 3 && $Y -ge 3 ]]; then
  echo "hello world"
fi


```

Reply:
> it should work at least for now, until Hadoop 3.4.0 is released

Rather until we start testing with the new docker image after Hadoop 3.4.0 release.  For that, we'll need to change this test script anyway, so I guess it's OK to defer until then.

Reply:
Yup I'm alright with the current approach. Thanks @jojochuang @adoroszlai